README.md

```diff
@@ -60,7 +62,95 @@ int axis = -1;
 KokkosFFT::rfft(execution_space(), x, x_hat, KokkosFFT::FFT_Normalization::BACKWARD, axis); // FFT along -1 axis and batched along 0th axis
 ```
 
-## Building KokkosFFT
+In this example, the 1D batched ```rfft``` over 2D View along ```axis -1``` is executed. Some basic examples are found in [examples](https://github.com/CExA-project/kokkos-fft/tree/main/examples).
+
+## Disclaimer
+KokkosFFT is under development and subject to change without warning. The authors do not guarantee that this code runs correctly in all the environments.
+
+## Using KokkosFFT
+For the moment, there are two ways to use KokkosFFT: including as a subdirectory in CMake project or installing as a library. First of all, you need to clone this repo.
+```bash
+git clone --recursive https://github.com/CExA-project/kokkos-fft.git
+```
 
 ### CMake
-Up to now, we just rely on the CMake options for Kokkos.
+Since KokkosFFT is a header-only library, it is enough to simply add as a subdirectory. It is assumed that kokkos and kokkosFFT are placed under ```<project_directory>/tpls```.
+Here is an example to use KokkosFFT in the following CMake project. 
+```
+---/
+ |
+ └──<project_directory>/
+    |--tpls
+    |    |--kokkos/
+    |    └──kokkosFFT/
+    |--CMakeLists.txt
+    └──hello.cpp
+```
+
+The ```CMakeLists.txt``` would be 
+```CMake
+cmake_minimum_required(VERSION 3.23)
+project(kokkos-fft LANGUAGES CXX)
+
+add_subdirectory(tpls/kokkos)
+add_subdirectory(tpls/kokkos-fft)
+
+add_executable(hello-kokkos-fft hello.cpp)
+target_link_libraries(hello-kokkos-fft PUBLIC Kokkos::kokkos KokkosFFT::fft)
+```
+
+For the compilation, we basically rely on the CMake options for Kokkos. For example, the configure options for A100 GPU is as follows.
+```
+cmake -DBUILD_TESTING=ON \
+      -DCMAKE_CXX_COMPILER=<project_directory>/tpls/kokkos/bin/nvcc_wrapper \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DKokkos_ENABLE_CUDA=ON \
+      -DKokkos_ENABLE_CUDA_CONSTEXPR=ON \
+      -DKokkos_ARCH_AMPERE80=ON \
+      -DKokkos_ENABLE_CUDA_LAMBDA=On ..
+```
+This way, all the functionalities are executed on A100 GPUs.
+
+### Install as a library
+Is is assumed that the Kokkos is installed under ```<lib_dir>/kokkos``` targeting Skylake with OpenMP backend. ```Kokkos_dir``` also needs to be set appropriately. Here is a recipe to install KokkosFFT under ```<lib_dir>/kokkosFFT```.
+
```

Comment:
Why Skylake ? I do not see anything specific with this architecture.

Reply:
Simply because I am using Icelake in my environment. Any CPU is fine, just an example.